lib/Installation/Partitioner/LibstorageNG/v4_3/FormatMountOptionsPage.pm

```diff
@@ -136,4 +139,46 @@ sub press_next {
     return $self->{btn_next}->click();
 }
 
+sub press_ok {
+    my ($self) = @_;
+    $self->{btn_ok}->click();
+}
+
+sub edit_fstab_options {
+    my ($self, $args) = @_;
+    $self->press_fstab_options();
+    $self->edit_fstab_mount_by($args);
+    $self->press_ok();
+}
+
+sub edit_fstab_mount_by {
+    my ($self, $args) = @_;
+    # The default value represents the mount_by value that we expect to be selected, before we edit it.
+    # If no arguement is definded, then "UUID" will be used as default value.
+    my $default            = $args->{default} || "UUID";
```

Comment:
"default" is what we expect to find pre-selected in the fstab mount_by option. Changing it, means we set a different value than UUID, to what we expect to find there. For example, on s390x the default value is not UUID. Would "preselected " be clearer name? 

I added the check for it because I thought it was in the scope of the ticket, based on the description, to check if by default, the system will use UUID as mount_by option. Plus, there is a follow-up ticket for extending the coverage, if case we use the existing partitioning (using a published image for installation) to see if the existing mount_by option is recognized by the partitioner : https://progress.opensuse.org/issues/89494 . 

Indeed, in this PR, changing the default is not used and I could just set it hard-coded. About the extra nesting, it looks much tidier indeed, but do we have limitations regarding the nesting depth?

Reply:
if you don't use it I would not include this logic yet, it requires more thinking about how to assert defaults. As you have it now default/preselected does not apply only to 'mount_by', and default could be written outside partition structure in yaml. I will default to UUID for now for x86_64 and have some offline team discussion about it.

Reply:
I was thinking what to do about this part. Honestly, it seem pretty simple to me. Setting for example
fstab_options:
  mount_by: UUID
  default_mount_by: UUID

would be clear enough and we would avoid extra nesting, but since I am not even using this feature here, I changed the code to hard coded value "UUID".